src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -426,6 +433,45 @@ private URI makeHistoryRangeURI(
     // build the final URI
     return builder.build();
   }
+
+  /**
+   * makeConfigureClientURI - Given a request UUID, and a fully qualified pipe name make a URI for
+   * the configure snowpipe client
```

Comment:
Can you add how would the URI look like. (Just copying from server side)
```suggestion
   * http://snowflakeURL{:PORT}/v1/data/pipes/{pipeName}/client/configure
   *
   * <p>Where snowflake URL can be an old url or new regionless URL.
   *
   * <p>Request Body is Empty
   *
   * <p>And our response looks like:
   *
   * <pre>
   *   {
   *      'clientSequencer': LONG
   *   }
   * </pre>

```

Reply:
Food for thought, should we return requestId from clientSDK? (I know we dont pass it down server side)
But since SDK is adding it, it would be good to return back requestId in response too.

Reply:
we can add them in the response. But in my understanding the requestId is only for debug purpose. However, when non 200 status code occurred, we don't return response object, instead we throw an error.  Should we add the requestId to IngestResponseException? If we want to return the requestId back to users, it makes sense we return them at both response and exception.